projects.csv

```diff
@@ -1,3 +1,3 @@
 path,name
 https://github.com/jhy/jsoup
-https://github.com/jhy/jsoup,secondJsoup
+https://github.com/jhy/jsoup,jsoupOptionalName
```

Comment:
@rafaelmotaalves this might be confusing. the following is better.

https://github.com/jhy/jsoup, jsoup
https://github.com/guilhermejccavalcanti/jFSTMerge
https://github.com/rbonifacio/soot-analysis